### ISSUE TITLE
Redesign Project Detail view

### ISSUE THREAD

**Issue by dennereed** [paleocore/paleocore#62]
Create a detail view for every project hosted on paleocore. Detail view has links to public view of data (public admin). Detail view has summary of project with abstract, number of records and map showing location of project site.

**Comment by dennereed** [comment]
See also issue #18

**Comment by wabarr** [comment]
I have a working prototype of this up on the paleocore_projects branch.  It looks pretty decent in my opinion, but could certainly use tweaking, and more content.

**Comment by dennereed** [comment]
The branch is actually called pcore_projects. It requires the following additional python libraries: 
django-leaflet == 0.15.2 by Leplatre
django-geojson == 2.6.0 by Leplatre

**Comment by dennereed** [comment]
Here's what I had to do to get the paleocore_projects app up and running. 

Checked out the new branch from GitHub called pcore_projects
Installed two new python libraries in my python virtual environment: 
django-leaflet == 0.15.2 by Leplatre
django-geojson == 2.6.0 by Leplatre
These are in addition to all the paleocore library dependencies documented on GitHub
Run syncdb to update my local development DB
Install new fixtures for taxonomy and mlp and drp from UT Box. 
Note: I tried creating a brand new DB and installing a complete fixture pulled from the production server, but I was unable to get dumpdata to work on the production DB. In the end I had to DROP all the drp, mlp and taxonomy tables and install just the two fixtures mentioned above.

With theses steps the new app runs.  However I still don't have a fixture for the project table so I have no projects visible on my map.
